### DISCUSSION TITLE
Unable to save HTTPX parameters as csv

### DISCUSSION THREAD

**Discussion by secfb** [projectdiscovery/httpx#639] [Q&A] [answered]
Due to the internal working, I need to check thousands of domains as follows.  I think the best way to check these domains is excel. HTTPX is a very nice tool, but I just can't output the parameters I want in csv format. Will there be a new feature for httpx in this regard? You can export in CSV format, but there are too many results.
```
cat 1.txt|httpx -ip -cname -title -status-code -tech-detect
```
<img width="801" alt="image" src="https://user-images.githubusercontent.com/38748801/170458774-e56156da-6e3a-448a-bceb-3a5cbbf8658b.png">

Result
```
https://testfire.net [200] [Altoro Mutual] [65.61.137.117] [Apache Tomcat,Java]
https://demo.testfire.net [200] [Altoro Mutual] [65.61.137.117] [Apache Tomcat,Java]
https://altoro.testfire.net [200] [Altoro Mutual] [65.61.137.117] [Apache Tomcat,Java]
https://www.testfire.net [200] [Altoro Mutual] [65.61.137.117] [testfire.net] [Apache Tomcat,Java]
```

**Comment by ehsandeep** [comment, marked as answer]
@secfb did you tried using `-csv` flag? 

```
echo hackerone.com | httpx -csv -silent -o output.csv
```
